frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/PipelineSuccessRatioDonut.tsx

```diff
@@ -36,6 +36,7 @@ const PipelineSuccessRatioDonut: React.FC<PipelineMetricsGraphProps> = ({
     delay: interval,
   });
   const pipelineSuccessData = runData?.data?.result ?? [];
+  const successString = 'Success';
```

Comment:
Any reason to capitalize this string? I don't see it being rendered anywhere. You wouldn't have to explicitly capitalize strings in the obj to check for it's equality if you just kept it `success`.

Reply:
It is used by the donut chart component on hover. I realized It could be capitalized directly inside the component itself. Updated.